proposals/xxxx-continous-integration.md

```diff
@@ -0,0 +1,73 @@
+# Configuration management
+
+* Proposal: [SDL-NNNN](xxxx-Configuration-Management.md)
+* Author: [Vadym Kushnirenko](https://github.com/vkushnirenko-luxoft)
+* Status: **Awaiting review**
+* Impacted Platforms: [Build server]
+
+## Introduction
+
+This proposal is about an order of creating build jobs in our buiding system, running tests and keeping results of theese tests.
+Also a good practice to provide a set of scripts to easy manage our build system.
+
+## Motivation
+
+The building process should be transparent. Any person should be able to deploy build server on his side, use it and propose recomendations about improvements.
+Build server configuration must be saved into files, be flexible and be applied in any moment from public resources.
+
+## Proposed solution
+
+Key components of build system are:
+- jenkins master - it manages build/test processes,
+- jenkins slaves - servers which builds and tests,
```

Comment:
@vkushnirenko-luxoft It's better to rephrase 'builds and tests' into something else, e.g. 'servers that allows to build SDL with different options and to run unit and ATF tests'

Reply:
Accept